styles.css

```diff
@@ -303,10 +303,17 @@ input[type="submit"] {
   min-width: 120px;
 }
 
+button:active,
+input[type="submit"]:active {
+  background-color: #2230d2;
+  box-shadow: none;
+  color: black;
+}
+
 button:hover,
 input[type="submit"]:hover {
   background-color: #6070ff;
-  color: white;
+  color: #fff;
```

Comment:
Your button hover state doesn't match the design, Kindly implement the hover to match the design. currently, both the `hover` and `active` states share the same color😉 and this doesn't match the design😉.

<img width="528" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/ce927a23-a440-474b-97f8-05cd7cd89b04">

when hovered
<img width="185" alt="image" src="https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/64643995/78b62b9a-fe05-4bb9-bc5c-67973db300a8">